packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -0,0 +1,37 @@
+{{#showBanner}}
+<div id="banner-brand-message" class="absolute w-full top-0 left-0">
+    <div class="flex justify-center p-2 brand-banner-bg">
+        {{#showPrelaunch}}
+        <div class="flex"/>
+            <div class="flex-none m-logo relative mt-1 mr-4 mb-2 bg-black">
+                <div>
+                    <img src="../../../images/brand_messaging/icon_m.png" alt="mozilla m logo" />
```

Comment:
Couple things on both images:
* Is this the right path? I see one other `img` reference in `post_verify/cad_qr` that references `src="/images/`.;
* I believe this alt text needs to go to l10n as well

Reply:
This did work, but I just updated to use b64 data strings.

Reply:
Alright, alt texts have been wrapped in `{{#t}}` or wrapped in `<Localized>`.